hw/ip/rstmgr/data/rstmgr.sv.tpl

```diff
@@ -98,24 +98,46 @@ module rstmgr
   logic local_rst_n;
   assign local_rst_n = resets_o.rst_por_io_div2_n[DomainAonSel];
 
-  logic [NumAlerts-1:0] alert_test, alerts;
+
   rstmgr_reg_pkg::rstmgr_reg2hw_t reg2hw;
   rstmgr_reg_pkg::rstmgr_hw2reg_t hw2reg;
 
+  logic reg_intg_err;
   rstmgr_reg_top u_reg (
     .clk_i,
     .rst_ni(local_rst_n),
     .tl_i,
     .tl_o,
     .reg2hw,
     .hw2reg,
-    .intg_err_o(alerts[0]),
+    .intg_err_o(reg_intg_err),
     .devmode_i(1'b1)
   );
 
+
+  ////////////////////////////////////////////////////
+  // Errors                                         //
+  ////////////////////////////////////////////////////
+
+  // consistency check errors
+  logic [${len(leaf_rsts)-1}:0][PowerDomains-1:0] const_chk_errs;
```

Comment:
Same renaming suggestion as for the file... const_chk_errs -> constc_chk_errs? Same below.

Reply:
fixed